spec/reek/report/location_formatter_spec.rb

```diff
@@ -0,0 +1,37 @@
+require_relative '../../spec_helper'
+require_relative '../../../lib/reek/report/formatter'
+require_relative '../../../lib/reek/smells/duplicate_method_call'
+require_relative '../../../lib/reek/smells/smell_warning'
```

Comment:
This is silly. Bundler will add ./lib to the load path, so we can just use `require` in this case.